packages/core/project.json

```diff
@@ -15,6 +15,12 @@
         "esbuildConfig": "esbuild.config.js"
       }
     },
+    "version": { "postTargets": ["core:github"] },
```

Comment:
@edbzn  is there a way to move`"postTargets": ["core:github"]` into `nx.json` target defaults with something like `^github` or so?

Reply:
No, it's not currently supported, but it's something I wanted to implement.